lib/commands/cache.js

```diff
@@ -298,18 +298,23 @@ class Cache extends BaseCommand {
       if (!this.npm.config.get('force')) {
         throw this.usageError('Please use --force to remove entire npx cache')
       }
+      const { npxCache } = this.npm.flatOptions
+      if (!this.npm.config.get('dry-run')) {
+        return fs.rm(npxCache, { recursive: true, force: true })
+      }
```

Comment:
@wraithgar I made an assumption that we want the whole folder gone in the force. That could be a bad assumption, let me know

Reply:
Right, if you `rm` with no args that removes everything, and that's why it's behind a `--force` guardrail.